engine/access/rest_api_test.go

```diff
@@ -136,22 +170,31 @@ func (suite *RestAPITestSuite) SetupTest() {
 		0,
 		false,
 		false,
-		nil,
-		nil,
 		suite.me,
+		suite.secureGrpcServer,
+		suite.unsecureGrpcServer,
 	)
 	assert.NoError(suite.T(), err)
 	suite.rpcEng, err = rpcEngBuilder.WithLegacy().Build()
 	assert.NoError(suite.T(), err)
 
 	suite.ctx, suite.cancel = irrecoverable.NewMockSignalerContextWithCancel(suite.T(), context.Background())
+
+	suite.secureGrpcServer.Start(suite.ctx)
+	suite.unsecureGrpcServer.Start(suite.ctx)
```

Comment:
in the builders, we startup the engine before the servers. We should do that here to verify the startup order works as expected.